__tests__/app.test.js

```diff
@@ -141,4 +138,36 @@ describe("Set of GET Tests", () => {
         });
     });
   });
+  describe("GET Comments By Article ID", () => {
+    test("should return all comments for a specific article_id", () => {
+      return supertest(app)
+        .get("/api/articles/5/comments")
+        .expect(200)
+        .then((response) => {
+          const comments = response.body;
+          expect(comments.length > 0).toBe(true);
+          expect(Array.isArray(comments)).toBe(true);
```

Comment:
This test needs to be more thorough. What are you expecting each comment object and their values to looks like? Its important to check the comment body to ensure its what you expect